### PR TITLE
Build docker images from docker-compose directly

### PR DESCRIPTION
**Pull request by svenvanderburg**
Fixes https://github.com/PerfectFit-project/virtual-coach-issues/issues/161

.

### CODE REVIEW

File: script/server [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
-bash fetch_and_build_all_images.sh
+bash script/bootstrap
```
Comment (dsmits) [inline script/server:2]: just run
```suggestion
script/bootstrap
```

File: script/test [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 pip install -r requirements-dev.txt
-bash script/fetch_and_build_all_images.sh
+bash script/bootstrap
```
Comment (dsmits) [inline script/test:3]:
```suggestion
script/bootstrap
```

**Review by dsmits — CHANGES_REQUESTED** [review]
Only two tiny suggestions

**Comment by svenvanderburg** [comment]
Thanks for the comments :)

**Pull request merged**